tests/playTests.sh

```diff
@@ -260,6 +260,11 @@ zstd tmp -c --compress-literals    --fast=1 | zstd -t
 zstd tmp -c --compress-literals    -19      | zstd -t
 zstd -b --fast=1 -i0e1 tmp --compress-literals
 zstd -b --fast=1 -i0e1 tmp --no-compress-literals
+println "test: --no-check for decompression"
```

Comment:
Do you want to try corrupting the input here? Something like [this](https://stackoverflow.com/a/5586379/1217213):

```
printf '\xDE\xAD\xBE\xEF' | dd of=tmp.zst bs=1 seek=$(($(wc -c tmp.zst) - 4)) count=4 conv=notrunc
```

Not sure if we're ok taking deps on `printf` and `wc`. I know we already depend on `dd`. @Cyan4973?

Or maybe:

```
head -c -4 < tmp.zst > tmp.corrupt.zst
echo -ne "\xDE\xAD\xBE\xEF" >> tmp.corrupt.zst
```

Reply:
The difficulty is remain compatible with `sh` in `playTests.sh`, 
which is more limited than `bash`.
But I believe that all formulations proposed are compatible.

The `freeBSD` CI test is supposed to help checking `sh` compatibility.